app/helpers/application_helper.rb

```diff
@@ -11,6 +11,24 @@ def login_link
 
   def logout_link
     return unless current_user
-    content_tag(:li, (link_to "Log Out", logout_path, method: 'delete'))
+    content_tag(:li, (link_to "Log Out", logout_path, method: "delete"))
+  end
+
+  def short_url_entry(form_obj)
+    return unless current_user
+    render partial: 'custom_url', object: form_obj, as: "f"
+  end
+
+  def show_link_box
+    return unless flash[:url]
+    render partial: 'link_box'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.